pkgs/applications/audio/foo-yc20/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, jack2, gtk2, lv2, faust, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "git-2015-05-21";
+  name = "foo-yc20-${version}";
+  src = fetchgit {
```

Comment:
It's generally faster and more efficient to use `fetchFromGitHub` when on github. They package up only the revision you ask for and you don't need to retrieve all of history and then discard most of it.